### ISSUE TITLE
[Feature Request] More robust UI

### ISSUE THREAD

**Issue by thecodrr** [madagaga/WTalk#6]
I was thinking of using [Mahapps.Metro](mahapps.com/). They are open-source and quite an active community. You should check them. 
If you agree, I can implement all the UI while you work on code-behind.

**Comment by madagaga** [comment]
I'm not fan of this UI. I prefer window 10 standard ui wich is provided in the SDK for UWP.
Check [Hangups-UWP](https://github.com/kfechter/Hangups-UWP) their UI is Standard windows 10

**Comment by thecodrr** [comment]
Yes, that is true but it can't be used with WPF, at least directly. We must use a toolkit that provides the Modern UI. You should look at Material Design of Google, it is being used in Android.
Windows 10 Metro UI can be made useful. The hamburger menu can be implemented but we should try something new.

**Comment by thecodrr** [comment]
Check this lib out: https://github.com/ButchersBoy/MaterialDesignInXamlToolkit
And this: https://github.com/dotMorten/UniversalWPF

**Comment by madagaga** [comment]
Yeah Nice !
But is it good to "fake" a system on another ? It sounds like having an osx
like app on windows. I don't know if it is "cool"

We just have to convert this app to Windows 10 universal app then we could use all the ui Framework. And it could run on mobile. 
Concerning material design i’ve been working with angular material which is an angular port of material design.
I don’t know what is the best solution

**Comment by thecodrr** [comment]
The downside of Universal Windows App is that it needs to be in the Windows Store for users to install it, which requires a license and the license further requires budget. Unless you have all that, it will be very easy to port it to UWP because UWP also uses C# .NET. Only the UI would change, the code-behind would remain the same. 
Why not maintain both? A WPF app for those users who are more comfortable with traditional interface and installation structure, and a UWP app for mobile and PC users. 

The way I see it, the core of WTalk can be used in both and will require little to no change. So it would be straight-forward to maintain both projects.

In the end, it is your decision :)

**Comment by madagaga** [comment]
It was the initial idea.
Having both UWP and desktop app.

Maybe you're right, we have to finish a desktop version with a cool UI.
Material Design is great but I prefer an UI which match with OS.
If I have a screen, or I can customize application theme to "match" to
windows 10 UI.

What do you think about this idea ?

**Comment by thecodrr** [comment]
Then [Mahapps.metro](https://github.com/MahApps/MahApps.Metro) will come in useful. There controls are exactly the same as Windows 10 UI, except a few. You should check them out. 

Your idea of customizing the application theme is good but it will require too much time and the result will not always be satisfactory. Some controls you cannot imitate. One way is to "copy" the applications theme from a UWP app :smile: You should try something out first.

**Comment by thecodrr** [comment]
But there is a problem. What if a user is running Windows 7? WTalk will look weird.

**Comment by madagaga** [comment]
You're right. Let's go for material design.

**Comment by thecodrr** [comment]
Good choice.
